sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -16,26 +30,26 @@
 
 ### New Features
```

Comment:
Merge beta.3 changes into beta.4, similar to https://github.com/Azure/azure-sdk-for-cpp/pull/978#discussion_r522599423 to get consistency with the releases.